pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-data-export</artifactId>
-  <version>3.1.0-SNAPSHOT</version>
+  <version>3.0.1</version>
```

Comment:
change this to 3.1.0-SNAPSHOT, other wise the release plugin will not work as it is not a snapshot version